revproxy/views.py

```diff
@@ -38,30 +45,57 @@ def __init__(self, *args, **kwargs):
 
     @property
     def upstream(self):
+        """This method is not implemanted on the base class """
```

Comment:
Typo here (implemanted).

Not sure if we need to document something so obvious like that.